agr_literature_service/api/schemas/reference_schemas.py

```diff
@@ -137,6 +138,7 @@ class ReferenceSchemaShow(BaseModelShow):
     open_access: Optional[bool] = None
     citation: Optional[str] = None
     workflow_tags: Optional[List[WorkflowTagSchemaShow]] = None
+    topic_entity_tag: Optional[List[TopicEntityTagSchemaShow]] = None
```

Comment:
topic_entity_tags instead of  topic_entity_tag.  But this also returns a '' instead of a [] so something else is wrong